src/manage.c

```diff
@@ -4071,6 +4071,7 @@ launch_osp_openvas_task (task_t task, target_t target, const char *scan_id,
 {
   osp_connection_t *connection;
   char *hosts_str, *ports_str, *exclude_hosts_str, *finished_hosts_str;
+  int alive_test = 0;
```

Comment:
For consistency, please keep initialisations out of the declaration block in gvmd.  Like how config and connection are done below.

I know some places in gvmd do it, but I've been trying to prevent new code from doing it because it got messy.